package.json

```diff
@@ -139,5 +139,8 @@
     ]
   },
   "author": "",
-  "license": "ISC"
+  "license": "ISC",
+  "dependencies": {
```

Comment:
this should not be in this package.json. it should be in the frontend package.json